pyproject.toml

```diff
@@ -34,3 +35,6 @@ golioth = "golioth.cli:main"
 [project.utils]
 "Homepage" = "https://github.com/golioth/golioth-zephyr-sdk"
 "Bug Tracker" = "https://github.com/golioth/golioth-zephyr-sdk/issues"
```

Comment:
Should these be updated to this repository?

Reply:
I'll make that change